docker-compose.deps.yml

```diff
@@ -27,7 +27,7 @@ services:
     extends:
```

Comment:
so `docker-compose -f docker-compose.deps.yml run --rm pip` now runs `poetry` maybe also change the name of the service?